src/inventory.cpp

```diff
@@ -29,6 +29,21 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "util/serialize.h"
 #include "util/string.h"
 
+//used in ItemStack::getRange()
+static float string_to_float(const std::string& str, bool& is_valid)
```

Comment:
We can use `std::optional` now that we have C++17 support, instead of having a boolean out parameter.